cfme/tests/v2v/test_v2v_migrations.py

```diff
@@ -225,3 +248,23 @@ def test_dual_datastore_dual_vm_mapping_crud(appliance, enable_disable_migration
     mapping = infrastructure_mapping_collection.create(form_data_dual_datastore)
     view = navigate_to(infrastructure_mapping_collection, 'All', wait_for_view=True)
     assert mapping.name in view.infra_mapping_list.read()
+
+
+@pytest.mark.parametrize('vm_list', ['NFS_Datastore_1', 'iSCSI_Datastore_1'], ids=['NFS', 'ISCSI'],
+                         indirect=True)
+@pytest.mark.parametrize('form_data_single_datastore', [['nfs', 'nfs']], indirect=True)
+def test_end_to_end_migration(appliance, enable_disable_migration_ui, vm_list, create_provider,
+                              form_data_single_datastore):
+    infrastructure_mapping_collection = appliance.collections.v2v_mappings
+    mapping = infrastructure_mapping_collection.create(form_data_single_datastore)
+    coll = appliance.collections.v2v_plans
+    coll.create(name="plan_{}".format(fauxfactory.gen_alphanumeric()),
+                description="desc_{}".format(fauxfactory.gen_alphanumeric()),
+                infra_map=mapping.name,
+                vm_list=vm_list,
+                start_migration=True)
+    view = appliance.browser.create_view(navigator.get_class(coll, 'All').VIEW)
+    # explicit wait for spinner of in-progress status card
+    wait_for(lambda: bool(view.progress_bar.is_plan_started(coll.name)),
```

Comment:
@Yadnyawalkya this line is not working for me. It is failing to find the `view`.